10-Sprint-Mission/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/style.css">
+  <title>판다마켓</title>
+</head>
+<body>
+  <header class="header">
+    <a href="index.html">
+      <img src="./img/lg/Property 1=md.png" alt="로그인">
+    </a>
+    <a href="./links/login.html">
+      <img src="./img/btn/btn_small.png" alt="로그인버튼">
+    </a>
+  </header>
+  <div>
+    <section class="top_banner">
+      
+        <button class="btn top">
+          <p>일상의 모든 물건을 <br>거래해보세요</p>
+          <a href="./links/item.html">
+          <img class="itemlink" src="./img/btn/state=active.png" alt="구경하러가기">
+          </a>
+        </button>
+        <img src="./img/Img_home_top.png" alt="배너">
+      
+    </section>
+  </div>
+  <div>
+    <section class="main-container">
+      <section class="main-item">
+        <div class="image-container">
+          <img src="./img/main/Img_home_01.png" alt="상">
+        </div>
+        <div class="text-container">
+          <h5>Hot Item</h5>
+          <p class="focus">인기 상품을<br> 확인해 보세요</p>
+          <p class="focus small">가장 HOT한 중고거래 물품을<br> 팬다 마켓에서 확인해 보세요</p>
+        </div>
+      </section>
+      <section class="main-item middle">
+        <div class="text-container middle">
+          <h5>Search</h5>
+          <p class="focus">구매를 원하는<br> 상품을 검색하세요</p>
+          <p class="focus small">구매하고 싶은 물품은 검색해서<br> 쉽게 찾아보세요</p>
+        </div>
+        <div class="middle">
+          <img src="./img/main/Img_home_02.png" alt="중">
+        </div>
+      </section>
+    
+      <section class="main-item">
+        <div class="image-container">
+          <img src="./img/main/Img_home_03.png" alt="하">
+        </div>
+        <div class="text-container">
+          <h5>Register</h5>
+          <p class="focus">판매를 원하는<br> 상품을 등록하세요</p>
+          <p class="focus small">어떤 물건이든 판매하고 싶은 상품을 쉽게 등록하세요</p>
+        </div>
+      </section>
+    </section>
+  </div>
+  <div>
+    <section class="bottom_banner">
+      <img src="./img/Img_home_bottom.png" alt="하단배너">
+    </section>
+  </div>
+  <footer class="foot">
+    <section class="foot-container">
+      <section class="foot-left">©codeit - 2024</section>
+      <section class="foot-center">
+        <a href="links/Privacy.html">Privacy Policy</a>
+        <a href="links/faq.html">FAQ</a>
+      </section>
+      <section class="foot-right">
+        <a href="https://www.facebook.com/"><img src="/img/icons/ic_facebook.png" alt="facebook"></a>
```

Comment:
P3:
외부영역으로 이동이 되는 경우는 새창으로 열어도 좋을 것 같습니다.
```suggestion
        <a href="https://www.facebook.com/" target='_blank'><img src="/img/icons/ic_facebook.png" alt="facebook"></a>
```